flask_server.py

```diff
@@ -85,12 +84,18 @@ def settings_page():
             if setting_name == "extra_ids":
                 extra_ids = request.form["extra_ids"].splitlines()
                 settings.update("extra_ids", extra_ids)
+            elif setting_name in ["loop_delay", "loop_variation"] and len(request.form[setting_name]) != 0:
+                settings.update(setting_name, int(request.form[setting_name]))
             elif len(request.form[setting_name]) != 0:  # Updates other non-null settings.
                 settings.update(setting_name, request.form[setting_name])
```

Comment:
As we start to have more settings maybe it's better if we update them in a in memory copy and save them to file later! 
Calling settings.update every time it;'s not very efficent.

Reply:
Done